src/_premake_init.lua

```diff
@@ -268,6 +286,22 @@
 	}
 
 	api.register {
+		name = "debugtoolargs",
+		scope = "config",
+		kind = "list:string",
+		tokens = true,
+		pathVars = true,
+	}
+
+	api.register {
+		name = "debugtoolcommand",
+		scope = "config",
+		kind = "path",
+		tokens = true,
+		pathVars = true,
+	}
```

Comment:
I have the same feeling, but I can't think of a simplification that both 1) doesn't significantly change the long-term api's, and 2) expresses everything I need.
I think part of it is just a misconception that debug configuration should be relatively simple, but it turns out it's actually really not. There's a ton of stuff that needs to be configured properly for it to work in any situation. A lot of these options are niche, and wouldn't usually be touched. Most of them can have automated defaults. Perhaps I could just automate the outputs directly, rather than have api's at all... but I think it's better to create an API and assign it a default, that way it can be overridden in the inevitable case where it's necessary.

If you're happy with this for now, that'd be good, and I'm more than happy to rearrange it in the future if we can pin down a simpler set of api's that are able to express everything.